gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/modules/restli/FlowConfigsV2ResourceHandler.java

```diff
@@ -164,16 +165,14 @@ public UpdateResponse  partialUpdateFlowConfig(FlowId flowId,
     return updateFlowConfig(flowId, flowConfig, modifiedWatermark);
   }
 
-  public UpdateResponse updateFlowConfig(FlowId flowId,
-      FlowConfig flowConfig) throws FlowConfigLoggedException {
+  public UpdateResponse updateFlowConfig(FlowId flowId, FlowConfig flowConfig) throws FlowConfigLoggedException {
     // We have modifiedWatermark here to avoid update config happens at the same time on different hosts overwrite each other
     // timestamp here will be treated as largest modifiedWatermark that we can update
     long version = System.currentTimeMillis() / 1000;
     return updateFlowConfig(flowId, flowConfig, version);
   }
 
-  public UpdateResponse updateFlowConfig(FlowId flowId,
-      FlowConfig flowConfig, long modifiedWatermark) throws FlowConfigLoggedException {
+  private UpdateResponse updateFlowConfig(FlowId flowId, FlowConfig flowConfig, long modifiedWatermark) throws FlowConfigLoggedException {
```

Comment:
surprised to see this private when the interface had this public

Reply:
no, it did not exist in older `FlowConfigsResourceHandler` interface. it could be private so changed it to private

Reply:
sorry, I missed that this overloads the method that IS in the interface, but just w/o the `long modifiedWatermark`